docs/src/examples/connectivity/index.mdx

```diff
@@ -0,0 +1,127 @@
+---
+name: Connectivity
+menu: Examples
+route: /examples/connectivity
+---
+
+# Connectivity
+
+Applications require notifying users when the state of their connection changes. One way of doing so is to make use of snackbars to inform users as shown in [this article](https://developers.google.com/web/fundamentals/instant-and-offline/offline-ux#inform_users_of_their_current_state_and_change_of_state).
+In this example, we see how this can be accomplished by making use of reactions.
+
+## Install the connectivity plugin
+
+The first step is to install the [`connectivity` plugin](https://pub.dev/packages/connectivity) available from [pub](https://pub.dev). Your application's `pubspec.yaml` file will look similar to the following
+
+```
+dependencies:
+  connectivity:
+```
+
+## The `ConnectivityStore`
+
+We now need to define our `Store` to hold our reactive state that represents the connection status of the device being used to run the application.
+
+```dart
+import 'package:mobx/mobx.dart';
+import 'package:connectivity/connectivity.dart';
+
+part 'connectivity_store.g.dart';
+
+class ConnectivityStore = _ConnectivityStore with _$ConnectivityStore;
+
+abstract class _ConnectivityStore with Store {
+  _ConnectivityStore() {
+    connectivityStream = ObservableStream(Connectivity().onConnectivityChanged);
+  }
+
+  @observable
+  ObservableStream<ConnectivityResult> connectivityStream;
+}
+```
+
+The `ConnectivityStore` has a field named `connectivityStream`. It is an `ObservableStream` that wraps around a stream provided by the `connectivity` plugin that can be used to listen for changes in connection state.
+By doing so, we can make use of reactions to respond to changes as we shall soon see. Without MobX, this would generally have been done by [listening to the stream](https://api.dart.dev/stable/dart-async/Stream/listen.html).
+
+## Setting up the reaction
+
+To have our user interface inform users of changes about the state of their connection, we'll need to configure a reaction.
+We'll need to '`StatefulWidget` so that the reaction can be configured within the `initState` method and disposed of when the state itself is disposed. The complete code is as follows
```

Comment:
Minor grammatical errors: 

- "We'll need to use a StatefulWidget..." 
- disposed off (double f)

Reply:
done. thanks for picking that up

Reply:
Oh noticed you wrote "disposed off". I haven't seen anyone use that in this context here in Australia. A search online showed one result where someone was taught that in India. See https://forum.wordreference.com/threads/dispose-of-dispose-off.556486/

Reply:
I've reworded it so this shouldn't be as relevant anymore :)